app/src/main/res/values/strings.xml

```diff
@@ -151,6 +151,55 @@
+
+    <!-- Action for restoring the default list of search engines -->
+    <string name="preference_search_restore">Restore default search engines</string>
+
+    <!-- Action for adding an additional search engine to the list -->
+    <string name="preference_search_add">Add another search engine</string>
+    <string name="preference_search_remove">Remove</string>
+
+    <!-- Clickable text in tutorial to open a dialog to manually add a search engine -->
+    <string name="tutorial_search_manually_add">MANUALLY ADD</string>
+    <string name="tutorial_search_cancel">CANCEL</string>
+
+    <string name="tutorial_search_title">Add search engine</string>
+    <string name="tutorial_search_step1">1. Go to the search engine homepage.</string>
+    <string name="tutorial_search_step2">2. Tap into the search engine\'s search field.</string>
+    <!-- An alternative string for Step 2 of the tutorial -->
+    <string name="tutorial_search_step2_alternative">2. Long press the search engine\'s search field.</string>
+    <string name="tutorial_search_step3">3. Tap the magnifying glass icon above the keyboard to add the search engine.</string>
+    <!-- Clickable action text that shows when user long-presses on a search engine search field (this is similar to paste or copy when long-pressing a text field) -->
+    <string name="action_option_add_search_engine">ADD SEARCH ENGINE</string>
+
+    <!-- An alternative string for Step 3 of the tutorial. Parameter 1 is &action_option_add_search_engine, which is the action text that the tutorial is referring to -->
+    <string name="tutorial_search_step3_alternative">3. Tap %1$s</string>
+    <string name="tutorial_search_step4">4. If you don’t see the magnifying glass, try manually adding a search engine.</string>
+
+    <!-- An alternative string for Step 4 of tutorial. Instruction with suggested action to add search engine, and alternative if that fails. Parameter 1 is &action_option_add_search_engine, which is the clickable action text for adding a search engine. -->
+    <string name="tutorial_search_step4_alternative">4. If you don’t see %1$s, try manually adding a search engine.</string>
+    <string name="tutorial_search_finish">GET STARTED</string>
+
+    <string name="search_add_hint">Tap the magnifying glass icon above the keyboard to add this search engine.</string>
+
+    <string name="search_add_manually_name">Name to display</string>
+    <string name="search_add_manually_name_hint">Search engine name</string>
+    <string name="search_add_manually_string">Search string to use</string>
+    <string name="search_add_manually_save">Save</string>
+
+    <!-- Explanation of how to add search string, with instruction on how to modify the string if necessary. The string "%s" should not be changed or removed. -->
+    <string name="search_add_manually_string_hint">Paste or enter search string. If necessary, replace search term with: %s.</string>
```

Comment:
Here (and for the string below) I wonder whether Android will understand %s as a placeholder and whether we need to escape it somehow. I'd test this quickly in a layout.

Reply:
I tested this in Focus quickly and it works -  it looks like as long as the %s isn't in the same string as other formatted strings (%1$s) it won't cause problems.